babel.config.js

```diff
@@ -39,7 +37,15 @@ const config = {
 			),
 			{ async: isBrowser && codeSplit },
 		],
-		[ path.join( __dirname, 'server', 'bundler', 'babel', 'babel-lodash-es' ), { from, to } ],
+		isBrowser && [
+			'module-resolver',
+			{
+				alias: {
+					lodash: 'lodash-es',
+					'lodash/': ( [ , name ] ) => `lodash-es/${ name }`,
```

Comment:
do we have any `lodash.*` in deps? like `lodash.map`?

Reply:
Doesn't look like it. I only see `lodash-es` in the bundle analysis, no `lodash` or `lodash.map`.

There's a fair bit of `lodash-es` function duplication, presumably due to WebPack's rules. `kebabCase` is in 18 chunks, for example, but it's 148B uncompressed.